src/runner.js

```diff
@@ -256,21 +264,34 @@ class Runner {
     let request;
     if (_request.protocol !== 'http:') {
       request = https.request(_request);
+      request.once('socket', (s) => {
+        s.once('secureConnect', function secureConnectOnce() {
+          let session = s.getSession();
+          let sessionId = session.slice(17, 17+32).toString('hex');
```

Comment:
Some explanation of what is going on here and what these numbers are would be nice.